docs/main/android/troubleshooting.md

```diff
@@ -78,6 +78,18 @@ The solution is to remove any old apps and make sure your package name is up to
 
 Finally, do a clean and rebuild just in case.
 
+## Error: "Unable to locate a Java Runtime"
+
+This error may occur when using the `run` command if the `JAVA_HOME` environment variable is not set on a Mac device. 
```

Comment:
This is not specific of Mac computers, can also happen on Windows, can you also document it for windows?

Reply:
Updated!